GCC 14 made implicit function definitions an error and the forward declaration
for display_state() was using the wrong name, causing a compilation
failure at the call site.

Upstream: Submitted to author via email
Signed-off-by: Florian Larysch <fl@n621.de>

diff -purN old/nanocom.h new/nanocom.h
--- old/nanocom.h	2025-08-10 01:34:12.000284003 +0200
+++ new/nanocom.h	2025-08-10 01:27:59.545821909 +0200
@@ -56,7 +56,7 @@ void cleanup_termios(int signal);
 void init_stdin(struct termios *sts);
 void init_comm();
 void usage( char *str);
-void display_status();
+void display_state();
 
 
 /*typedef to represent the current line state
